app/controllers/sessions_controller.rb

```diff
@@ -1,4 +1,42 @@
 class SessionsController < ApplicationController
-  def new
+  # layout without_logout
+    layout false
+    skip_before_action :access_control, :redirect_unauthenticated,
+                       :check_primary_account
+
+    def destroy
+      session.clear
+      if Rails.env.production?
+        redirect_to '/Shibboleth.sso/Logout?return=https://webauth.umass.edu/Logout'
+      else redirect_to dev_login_path
+      end
+    end
+
+    def dev_login # route not defined in production
+      if request.get?
+        @admin = User.where(admin: true)
+        @user  = User.where(admin: false)
```

Comment:
I was thinking of changing it to `dispatcher` ... better or worse? Technically admins are also dispatchers, but they're a special type of dispatcher.